web/frontend/src/components/slam/2d-legend.vue

```diff
@@ -0,0 +1,65 @@
+<template>
```

Comment:
Where is this 2D legend in your images attached to this PR?

Reply:
I've moved the preexisting 2d renderer legend that shows the direction of `x,y` into its own component. In general the JS community considers breaking down components into the smallest possible unit to be best practice. It makes refactoring and testing easier, and usually creates a clearer separation of concerns. This is a great example: this code for the legend is able to be entirely isolated and abstracted away into a `<Legend>` component.

Reply:
I was more asking about where the 2d legend is in the images attached. I dont see it in the video/image you provided.

Reply:
Oh, it's just the existing one for the 2d renderer:
<img width="51" alt="Screenshot 2023-05-22 at 14 42 19" src="https://github.com/viamrobotics/rdk/assets/103450731/2cdfe879-4a15-4676-a52c-5f10c3e380d2">